requirements.txt

```diff
@@ -4,7 +4,7 @@ hypothesis==3.2
 matplotlib>=2.0.0,<3.0.0
 numpy>=1.9.2
 pandas>=0.18.1
-prompt-toolkit>=1.0.7,<2.0.0
+prompt-toolkit>=1.0.7
```

Comment:
This breaks the system install of ipython which is not yet compatible with `prompt-toolkit>2.0.0`. https://github.com/Axelrod-Python/Axelrod/issues/1183

Once ipython is using the latest `prompt-toolkit` we will migrate to just use `2.0.0+`.

Reply:
See also: https://github.com/Axelrod-Python/Axelrod/issues/1217

Reply:
IPython already requires v2+: https://github.com/ipython/ipython/blob/7.1.1/setup.py#L192

Reply:
Actually, the oldest IPython that required prompt_toolkit v1 was the 6.5.0, and the marked line is compatible with it (actually, it's compatible with both versions). https://github.com/ipython/ipython/blob/6.5.0/setup.py#L193

Reply:
Perhaps my diagnosis of the source of the issue of #1217 and the linked discussions on #1183 isn't correct. Nonetheless the fix in #1212 is preferable as it'll move us to just using `prompt_toolkit` v1 **but** any fix needs to not break the common data science tools like jupyter and ipython and (as of 3 minutes ago): updating to the conda distribution of Jupyter currently keeps ipython at `6.5`.